nf_core/__main__.py

```diff
@@ -157,7 +157,7 @@ def nf_core_cli(verbose, log_file):
     default="release",
     help="How to sort listed pipelines",
 )
-@click.option("--json", is_flag=True, default=False, help="Print full output as JSON")
+@click.option("--json", is_flag=True, default=False, help="Print unfiltered list as JSON")
```

Comment:
@vbarrera , could you please elaborate why `unfiltered` should be used in this context?